Resources/Prototypes/Entities/Structures/Piping/Atmospherics/pipes.yml

```diff
@@ -71,6 +71,7 @@
     range: 2
     sound:
       path: /Audio/Ambience/Objects/gas_hiss.ogg
+  - type: GasAnalyzable
 
```

Comment:
Should probably also add this to gas canisters and tanks

Reply:
This is disabled for gas canisters because it shows the pressure/temperature of the output port, not the contents, as one might expect. Also, it is very easy to read off the pressure inside a canister simply by opening the UI.

This does not work for gas tanks, because they are not part of a pipe net.